RELEASES.md

```diff
@@ -0,0 +1,112 @@
+        * [Fix the networking address reusing behavior (`SO_REUSEADDR`)](https://github.com/asterinas/asterinas/pull/2277)
+    * Security:
+        * [Add basic cgroupfs implementation](https://github.com/asterinas/asterinas/pull/2121)
+* New device support:
+    * [Add basic i8042 keyboard support](https://github.com/asterinas/asterinas/pull/2054)
+* Enhanced device support:
+    * TTY
+        * [Refactor the TTY abstraction to support multiple I/O devices correctly](https://github.com/asterinas/asterinas/pull/2108)
+        * [Enhance the framebuffer console to support ANSI escape sequences](https://github.com/asterinas/asterinas/pull/2210)
+* Test infrastructure:
+    * [Introduce the system call tests from LTP](https://github.com/asterinas/asterinas/pull/2053)
+    * [Use Nix to build initramfs](https://github.com/asterinas/asterinas/pull/2101)
+
+## OSTD & OSDK
+
+We have made the following key changes to OSTD:
+
+* CPU architectures:
+    * x86-64:
+        * [Refactor floating-point context management in context switching and signal handling](https://github.com/asterinas/asterinas/pull/2219)
+        * [Use iret instead of sysret if the context is not clean](https://github.com/asterinas/asterinas/pull/2271)
+        * [Don't treat APIC IDs as CPU IDs](https://github.com/asterinas/asterinas/pull/2091)
+        * [Fix some CPUID problems and add support for AMD CPUs](https://github.com/asterinas/asterinas/pull/2273)
+    * RISC-V:
+        * [Add RISC-V timer support](https://github.com/asterinas/asterinas/pull/2044)
+        * [Parse device tree for RISC-V ISA extensions](https://github.com/asterinas/asterinas/pull/2113)
+    * LoongArch:
+        * [Add the initial LoongArch support](https://github.com/asterinas/asterinas/pull/2260)
+* CPU:
+    * [Add support for dynamically‌-allocated CPU-local objects](https://github.com/asterinas/asterinas/pull/2036)
+    * [Require `T: Send` for `CpuLocal<T, S>`](https://github.com/asterinas/asterinas/pull/2171)
+* Memory management:
+    * [Adopt a two-phase locking scheme for page tables](https://github.com/asterinas/asterinas/pull/1948)
+* Trap handling:
+    * [Create `IrqChip` abstraction](https://github.com/asterinas/asterinas/pull/2107)
+* Task and scheduling:
+    * [Rewrite the Rust doc of OSTD's scheduling module](https://github.com/asterinas/asterinas/pull/2284)
+    * [Fix the race between enabling IRQs and halting CPU](https://github.com/asterinas/asterinas/pull/2052)
+* Test infrastructure:
+    * [Add CI to check documentation and publish API documentation to a self-host website](https://github.com/asterinas/asterinas/pull/2218)
+
+We have made the following key changes to OSDK:
+
+* [Add OSDK's code coverage feature](https://github.com/asterinas/asterinas/pull/2203)
+* [Support `cargo osdk test` for RISC-V](https://github.com/asterinas/asterinas/pull/2168)
+
+## Contributors
```

Comment:
I think it is kind of off the topic to include contributors in the `RELEASES.md` file. I am inclined to remove it from this file but keep it in the "Announcing Asterinas 0.16.0" blog post, which seems to be a more appropriate place.

Reply:
Yes, I agree with removing the contributors from this file. I also think we should include the email address for each contributor in the blog post. Names alone are not reliable identifiers (I assume the names below come from git commit messages rather than their GitHub usernames, which can differ), but emails should be accurate.

Reply:
But publicizing emails would raise privacy concerns and may lead to more span emails. So I don't think using emails is a good idea.

Reply:
Thanks for  your explanation! It makes sense to not include email address for privacy concerns.